openlibrary/templates/lib/nav_head.html

```diff
@@ -2,12 +2,7 @@
 
 <header id="header-bar" class="header-bar">
   <div class="hamburger-component checkbox-menu">
-
-    <label for="toggle-hamburger" class="hamburger-button">
-      <img src="/static/images/menu.png" alt="additional options menu"/>
-    </label>
-
-    <input role="button" type="checkbox" class="hamburger-component__checkbox" id="toggle-hamburger">
+    <button type="button" class="hamburger-button"><img src="/static/images/menu.png" alt="additional options menu"/></button>
```

Comment:
Similar to the elements above, we'll need to add the `aria-expanded` attribute here as well. No need for the `role` because this is already a button 😄